src/Swashbuckle.AspNetCore.SwaggerGen/DependencyInjection/SwaggerGenOptionsExtensions.cs

```diff
@@ -422,7 +422,9 @@ public static void SupportNonNullableReferenceTypes(this SwaggerGenOptions swagg
         /// <param name="xmlDocFactory">A factory method that returns XML Comments as an XPathDocument</param>
         /// <param name="includeControllerXmlComments">
         /// Flag to indicate if controller XML comments (i.e. summary) should be used to assign Tag descriptions.
-        /// Don't set this flag if you're customizing the default tag for operations via TagActionsBy.
+        /// If customizing the default tag for operations via TagsAttribute, only the first Tag per controller will be
+        /// associated with the description.
+        /// However, don't set this flag if you're customizing the default tag for operations via TagActionsBy.
```

Comment:
Let's make `TagsAttribute` and `TagActionsBy` `<see .... />` references in these two comments.